data/hai/unfettered side missions.txt

```diff
@@ -65,3 +66,403 @@ mission "Unfettered returning home"
 		"reputation: Hai Merchant" += 10
 		"reputation: Hai Merchant (Human)" += 10
 		"reputation: Hai Merchant (Sympathizers)" += 10
+
+
+
+# This mission will offer but defer itself the first time if you lack money but have sufficient net worth.
+mission "Hai Legacy 1"
+	invisible
+	shipyard
+	source "Hai-home"
+	destination "Firelode"
+	clearance
+	to offer
+		has "First Contact: Unfettered: offered"
+		"reputation: Hai (Unfettered)" < 0
```

Comment:
Why less than zero?

Reply:
that's what it takes to be hostile to them

this mission offers before you fix your rep using JD, as an alternative path